exercism/bob/bob.spec.js

```diff
@@ -8,60 +8,60 @@ describe('Bob', function() {
     expect(result).toEqual('Whatever.');
   });
 
-  xit('shouting', function() {
```

Comment:
I cloned your project so I could test it and give detailed feedback. It's looking great, with clean code, though it's difficult for me to test since many of the tests have been skipped.